core/src/test/java/org/apache/james/mime4j/stream/RawFieldParserTest.java

```diff
@@ -461,4 +461,37 @@ public void testRawBodyParseFolded() {
         org.junit.Assert.assertEquals("simple boundary", params.get(0).getValue());
     }
 
+    @Test
+    public void testRegressionForContentDispositionParsingASCIIonly() {
+        ByteSequence buf = ContentUtil.encode(
+                "Content-Disposition: form-data; name=\"filedata\"; filename=\"Sanity a.doc\"");
```

Comment:
For parsing just parameters this should be:

```suggestion
                "name=\"filedata\"; filename=\"Sanity a.doc\"");
```